draft-pauly-dprive-oblivious-doh.md

```diff
@@ -164,20 +170,28 @@ Clients MUST set the HTTP Content-Type header to "application/oblivious-dns-mess
 to indicate that this request is an Oblivious DoH query intended for proxying. Clients
 also SHOULD set this same value for the HTTP Accept header.
 
-Proxies MUST check that Client requests are correctly encoded, and MUST return a
-4xx (Client Error) if the check fails, along with the Proxy-Status response header
-with an "error" parameter of type "http_request_error" {{!I-D.ietf-httpbis-proxy-status}}.
 A correctly encoded request has the HTTP Content-Type header "application/oblivious-dns-message",
 uses the HTTP POST method, and contains "targethost" and "targetpath" variables.
+The Proxy MUST validate these parameters and construct a URI Template as follows:
 
-The "targethost" and "targetpath" variables are used to construct the request to forward to
-the Target. The Proxy is expected to send the Client's request using the URI
-constructed as "https://targethost/targetpath".
+1. Let $TARGET be the "targethost" parameter from the Client's request, encoded as the
+concatenation of a "host" value (Section 3.2.2 of {{!RFC3986}}) and, optionally, the
```

Comment:
This encoding shouldn't be phrased as a Proxy requirement since it applies to both Client and Proxy, I'd move this closer to the URI template definition

Reply:
Fixed.